man/org.freedesktop.systemd1.xml

```diff
@@ -2761,6 +2767,7 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       readonly s Result = '...';
       readonly s ReloadResult = '...';
       readonly s CleanResult = '...';
+      readonly s MountResult = '...';
```

Comment:
hmm, I think we should follow a systematic naming scheme here, i.e. if the boolean prop above is called CanXYZ, then this field here should be called XYZResult too. But right now one uses "LiveMount", and the other just "Mount"

Reply:
just "mount" seems prone to confusion with mount units or systemd-mount, so I'll go the other way around and rename to LiveMountResult